lib/docker-sync/preconditions.rb

```diff
@@ -43,4 +43,12 @@ def self.unison_available
       raise('Could not find unison binary in path. Please install it, e.g. using "brew install unison"')
     end
   end
+
+  def self.unox_available
+    if (find_executable0 'unison-fsmonitor').nil?
+      raise('Could not find unison-fsmonitor binary in path. Please install it, see https://github.com/hnsl/unox, or simply run :
+      wget -O /usr/local/bin/unison-fsmonitor https://raw.githubusercontent.com/hnsl/unox/master/unox.py \
+      && chmod +x /usr/local/bin/unison-fsmonitor')
+    end
```

Comment:
@mickaelperrin a pitty there is now brew alternatives. I am thinking about creating a own tap and adding this.

Actually, in this case, use yes? and ask if the command should be run, run it automatically then. This makes the most sense here to ease up the installation